src/shared/components/cohort/Icon.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import DefaultTooltip from "../DefaultTooltip";
+
+export interface IIconProps
+{
+    text: string;
+    tooltip?: JSX.Element;
+    stroke?: string;
+    circleFill?: string;
+    fontSize?: number;
+    textFill?: string;
+}
+
+/**
+ * @author Selcuk Onur Sumer
+ */
+export default class Icon extends React.Component<IIconProps, {}>
```

Comment:
Good idea to separate this out. It may end up coming in handy some other time too, it's a nice little visualization. But `Icon` seems too general for what this is. What do you think of `LetterIcon`?